packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -164,9 +161,20 @@ const OrderDetails = observer(({ onPageReturn }) => {
                         {is_active_order &&
                             (order_store?.has_order_payment_method_details ? (
                                 <div className='order-details-card--padding'>
-                                    <Text size='xs' weight='bold'>
-                                        {labels.payment_details}
-                                    </Text>
+                                    <section className='order-details-card__title'>
+                                        <Text size='xs' weight='bold'>
+                                            {labels.payment_details}
+                                        </Text>
+                                        <Button
+                                            className='p2p-my-ads__expand-button'
+                                            onClick={() => setShouldExpandAll(prev_state => !prev_state)}
+                                            transparent
+                                        >
+                                            <Text size='xss' weight='bold' color='red'>
+                                                {should_expand_all ? localize('Collapse all') : localize('Expand all')}
```

Comment:
The same could be applied here as I had previously mentioned in my other comment:

```
localize(`${should_expand_all ? 'Collapse all' : 'Expand all'}`)
```